diff --git a/node_modules/react-transition-group/cjs/Transition.js b/node_modules/react-transition-group/cjs/Transition.js
index b44cb76..c18d466 100644
--- a/node_modules/react-transition-group/cjs/Transition.js
+++ b/node_modules/react-transition-group/cjs/Transition.js
@@ -540,7 +540,9 @@ Transition.propTypes = process.env.NODE_ENV !== "production" ? {
    */
   timeout: function timeout(props) {
     var pt = _PropTypes.timeoutsShape;
-    if (!props.addEndListener) pt = pt.isRequired;
+    // reactstrap-date-picker patch
+    //    for removing warnings about null prop
+    //if (!props.addEndListener) pt = pt.isRequired;
 
     for (var _len = arguments.length, args = new Array(_len > 1 ? _len - 1 : 0), _key = 1; _key < _len; _key++) {
       args[_key - 1] = arguments[_key];
diff --git a/node_modules/react-transition-group/esm/Transition.js b/node_modules/react-transition-group/esm/Transition.js
index 986d29f..d8d037b 100644
--- a/node_modules/react-transition-group/esm/Transition.js
+++ b/node_modules/react-transition-group/esm/Transition.js
@@ -518,7 +518,10 @@ Transition.propTypes = process.env.NODE_ENV !== "production" ? {
    */
   timeout: function timeout(props) {
     var pt = timeoutsShape;
-    if (!props.addEndListener) pt = pt.isRequired;
+    
+    // reactstrap-date-picker patch
+    //    for removing warnings about null prop
+    //if (!props.addEndListener) pt = pt.isRequired;
 
     for (var _len = arguments.length, args = new Array(_len > 1 ? _len - 1 : 0), _key = 1; _key < _len; _key++) {
       args[_key - 1] = arguments[_key];
